porting/finalize/UBports-official-support/port-support-requirements.rst

```diff
@@ -0,0 +1,102 @@
+.. _PortSupportRequirements:
+
+Port support requirements
+==================================
+
+This document enumerates the requirements that must be met in order for devices to be deemed ship ready for UBports releases.
+To be considered ready, port maintainers MUST meet the requirements presented in this document, including any documents incorporated via reference.
+
+The use of “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” is per the IETF standard defined in :RFC:`2119`.
+
+-----
+
+Attaining exceptions
+********************
+
+Process
+-------
+
+* Device or software deviations from our core product (as described below) MUST be approved by the devrel team.
+* Device or software exceptions SHOULD be made via change request to this repository.
+* All device or software exceptions that are granted MUST be documented below at the `Ubuntu Touch device page`_ for all affected devices.
+
+Hardware support
+****************
+
+Audio
+-----
+
+* All devices MUST support audio playback for media content.
+* Phones MUST support in-call audio.
+* Phones MUST support speaker audio.
+* Tablet devices capable of in-call audio/speaker audio MUST support in-call/speaker audio.
+
+RIL
+---
+
+* All devices with RIL supported in their stock OS MUST support RIL for phone calls & data.
+* All devices with RIL supported in their stock OS SHOULD support emergency calling with a SIM inserted (112/911).
+* All devices with RIL supported in their stock OS SHOULD support emergency calling without a SIM inserted (112/911).
+* Data-only devices (defined as devices that have a RIL but do not support telephony stack due to hardware/firmware restrictions) are EXEMPTED from phone & emergency dialing requirements.
+
+Software support
+****************
+
+CVE
+---
+
+* Devices MUST support CVE patches for “high profile” exploits and vulnerabilities.
+* Devices SHOULD receive regular CVE patches to the device kernel and dependencies.
+
+Installer
+---------
+* Devices MUST support installation via `UBports Installer`_.
+
+Recovery
+--------
+
+* Devices MUST support resetting to factory defaults.
+* Devices MUST support updating via OTA.
+
+Quality of Life
+***************
+
+Commit authorship
+-----------------
+
+* All non-original commits MUST have proper authorship attribution from the source it was taken from or adapted from.
+
+Documentation
+-------------
+
+* The device MUST be listed at the `Ubuntu Touch device page`_; the device page needs to be kept up-to-date.
+
+Stability
+---------
+
+* Issues like the "screen of death" MUST NOT affect the device.
```

Comment:
Is this a specific issue? I don't really understand. Can we link smt here?

Reply:
This is (was?) known in the android community as a device, which is powered on but not responding to any input, like pressing power keys.

We can drop this and readd more points later on i think?

Reply:
ah. ok, yeah. Even after having attempted a port myself, I'm very detached from the android/xda lingo ... And I generally always found it too vague. I think it is better to keep this language out of the docs only only use it with air quotes and amended with technical descriptions. I think in this case it is fine to drop this ... or if you want to you add some more high level .... "It should turn on and actually allow me to see stuff on the screen, start apps and such " :D ... just kidding. I think it's fine as is now